masterlist.yaml

```diff
@@ -228,11 +233,16 @@ common:
   - &reqBetterMales
     <<: *reqModX
     subs: [ '[Better males - Beautiful nudes and faces - New hairstyles](http://www.nexusmods.com/skyrim/mods/2488)' ]
-
-  - &reqSkyUI # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
+  # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
+  - &reqSkyUI
     type: warn
     content: 'SkyUI.esp is not active in your load order. [SkyUI](http://www.nexusmods.com/skyrim/mods/3863) is required for this mod to be fully functional.'
     condition: 'not active("SkyUI.esp")'
+  - &reqRequiemPatch
+    type: warn
+    content:
+      - lang: en
+        text: 'Incompatible with: Requiem. You must install a compatibility patch.'
```

Comment:
I agree that it's not very helpful but that's somewhat the intention. Requiem patches are not a straight forward record patching and more about balancing. For some mods there are multiple patches by different authors or patches for different versions of Requiem. Sometimes linking to a specific patch may be misleading because said patch is slightly outdated or does something some users consider gameplay or immersion-breaking and for them it might be better not to use the mod at all.
I believe a generic message that points out incompatibility is better than linking all possible patches. The Requiem community is very helpful and google is a powerful friend. And it's far easier to maintain in the masterlist ;)